recipes/graalpy/build_graalpy.sh

```diff
@@ -0,0 +1,157 @@
+    cp -r $STANDALONE/* $PREFIX
+
+    if [ -z "${MACOS}" ]; then
+        # sulong ensures that the llvm toolchain uses libc++abi.so in the toolchain
+        # directory by dynamically making sure it's loaded in its toolchain wrappers,
+        # but the conda build cannot know this and complains
+        LIBCXX_RPATH=`patchelf --print-rpath $PREFIX/lib/llvm-toolchain/lib/*/libc++.so.1.0`
+        if [ -n "$LIBCXX_RPATH" ]; then
+            LIBCXX_RPATH="$LIBCXX_RPATH:\$ORIGIN"
+        else
+            LIBCXX_RPATH="\$ORIGIN"
+        fi
+        patchelf --set-rpath "$LIBCXX_RPATH" $PREFIX/lib/llvm-toolchain/lib/*/libc++.so.1.0
+    fi
+
+    # match cpython include folder structure
+    ln -sf $PREFIX/include $PREFIX/include/python${PY_VERSION}
+
+    # license is packaged by the build process
+    cat $PREFIX/LICENSE_GRAALPY.txt $PREFIX/THIRD_PARTY_LICENSE_GRAALPY.txt > $SRC_DIR/LICENSE_GRAALPY.txt
+    rm $PREFIX/LICENSE_GRAALPY.txt $PREFIX/THIRD_PARTY_LICENSE_GRAALPY.txt
+else
+    GRAALVM=`mx graalvm-home`
+    mkdir -p $PREFIX/lib/jvm
+    cp -r $GRAALVM $PREFIX/lib/jvm/
+    GRAALVM_PREFIX=$PREFIX/lib/jvm/${GRAALVM##*/}
+
+    # symlink binaries
+    mkdir -p $PREFIX/bin/
+    for i in $GRAALVM_PREFIX/bin/*; do
+        if [ -x "$i" ]; then
+            ln -sf "$i" $PREFIX/bin/
+        fi
+    done
+
+    # sulong ensures that the llvm toolchain uses libc++abi.so in the toolchain
+    # directory by dynamically making sure it's loaded in its toolchain wrappers,
+    # but the conda build cannot know this and complains
+    if [ -z "${MACOS}" ]; then
+        LIBCXX_RPATH=`patchelf --print-rpath $GRAALVM_PREFIX/lib/llvm/lib/*/libc++.so.1.0`
+        if [ -n "$LIBCXX_RPATH" ]; then
+            LIBCXX_RPATH="$LIBCXX_RPATH:\$ORIGIN"
+        else
+            LIBCXX_RPATH="\$ORIGIN"
+        fi
+        patchelf --set-rpath "$LIBCXX_RPATH" $GRAALVM_PREFIX/lib/llvm/lib/*/libc++.so.1.0
+    fi
+
+    # relativize rpath for jvm libs
+    if [ -z "${MACOS}" ]; then
+        # delete graalvm libraries not required for graalpy
+        rm $GRAALVM_PREFIX/lib/libsplashscreen.so
+        rm $GRAALVM_PREFIX/lib/libawt_xawt.so
+        rm $GRAALVM_PREFIX/lib/libjawt.so
+        rm $GRAALVM_PREFIX/lib/libfontmanager.so
+        rm $GRAALVM_PREFIX/lib/libjsound.so
+
+        for i in $GRAALVM_PREFIX/lib/*.so; do
+            SO_RPATH=`patchelf --print-rpath $i`
+            if [ -n "$SO_RPATH" ]; then
+                SO_RPATH="$SO_RPATH:\$ORIGIN/server"
+            else
+                SO_RPATH="\$ORIGIN/server"
+            fi
+            patchelf --set-rpath "$SO_RPATH" $i
+        done
+    fi
+
+    # delete linux bits from sulong's llvm toolchain
+    if [ -n "${MACOS}" ]; then
+        rm -rf $GRAALVM_PREFIX/lib/llvm/lib/clang/*/lib/linux/
+    fi
+
+    # match cpython include folder structure
+    mkdir -p $PREFIX/include/
+    ln -sf $GRAALVM_PREFIX/languages/python/include $PREFIX/include/python${PY_VERSION}
```

Comment:
This needs to be the other way. Delete `$GRAALVM_PREFIX/languages/python/include` and link from `$PREFIX/include/python${PY_VERSION}` to that file.

That's because we don't want the graalvm version embedded in file names of downstream packages.

Reply:
Done